NEWS

```diff
@@ -1,3 +1,29 @@
+Changes in Version 1.6.0
+
+  o Integrated work by Marius Appel on speeding up BFAST algorithms using Rcpp.
+  Whether the speedups are enabled is controlled by set_fast_options() vs
+  set_default_options(). The fast options are now default.
```

Comment:
We need to change this description once #9 is fixed (set_fallback_options).

Reply:
Also the readme!